packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -63,89 +64,60 @@
-            @row-mounted="rowMounted"
-            @file-click="triggerDefaultAction"
-            @file-dropped="fileDropped"
-            @sort="handleSort"
-          >
-            <template #contextMenuActions="{ resource }">
-              <context-actions :action-options="{ space, resources: [resource] }" />
-            </template>
-            <template #footer>
-              <pagination :pages="paginationPages" :current-page="paginationPage" />
-              <list-info
-                v-if="paginatedResources.length > 0"
-                class="oc-width-1-1 oc-my-s"
-                :files="totalFilesCount.files"
-                :folders="totalFilesCount.folders"
-                :size="totalFilesSize"
-              />
-            </template>
-          </resource-tiles>
-          <resource-details
-            v-else-if="displayResourceAsSingleResource"
-            :single-resource="paginatedResources[0]"
-            :space="space"
-          />
-          <resource-table
-            v-else
-            id="files-space-table"
-            v-model:selectedIds="selectedResourcesIds"
-            class="files-table"
-            :class="{ 'files-table-squashed': isSideBarOpen }"
-            :view-mode="viewMode"
-            :are-thumbnails-displayed="displayThumbnails"
-            :resources="paginatedResources"
-            :target-route-callback="resourceTargetRouteCallback"
-            :header-position="fileListHeaderY"
-            :drag-drop="true"
-            :sort-by="sortBy"
-            :sort-dir="sortDir"
-            :space="space"
-            @file-dropped="fileDropped"
-            @file-click="triggerDefaultAction"
-            @row-mounted="rowMounted"
-            @sort="handleSort"
-          >
-            <template #quickActions="{ resource }">
-              <quick-actions
-                :class="resource.preview"
-                class="oc-visible@s"
-                :space="space"
-                :item="resource"
-              />
-            </template>
-            <template #contextMenu="{ resource }">
-              <context-actions
-                v-if="isResourceInSelection(resource)"
-                :action-options="{ space, resources: selectedResources }"
-              />
-            </template>
-            <template #footer>
-              <pagination :pages="paginationPages" :current-page="paginationPage" />
-              <list-info
-                v-if="paginatedResources.length > 0"
-                class="oc-width-1-1 oc-my-s"
-                :files="totalFilesCount.files"
-                :folders="totalFilesCount.folders"
-                :size="totalFilesSize"
-              />
-            </template>
-          </resource-table>
+          <template v-else>
+            <resource-details
+              v-if="displayResourceAsSingleResource"
+              :single-resource="paginatedResources[0]"
+              :space="space"
+            />
+            <component
```

Comment:
I think here is a `v-else` missing, no? Or do we want `resource-details` to be displayed together with the table/tiles view?